Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -34,20 +34,24 @@ - (instancetype)initWithSettings:(NSDictionary *)settings andComScore:(id)scorAn
         SCORPublisherConfiguration *config = [SCORPublisherConfiguration publisherConfigurationWithBuilderBlock:^(SCORPublisherConfigurationBuilder *builder) {
             // publisherId is also known as c2 value
             builder.publisherId = settings[@"c2"];
-            builder.publisherSecret = settings[@"publisherSecret"];
-            builder.applicationName = settings[@"appName:"];
-            builder.usagePropertiesAutoUpdateInterval = [settings[@"autoUpdateInterval"] integerValue];
-            builder.secureTransmission = [(NSNumber *)[self.settings objectForKey:@"useHTTPS"] boolValue];
```

Comment:
Was `secureTransmission` supposed to be migrated to the general config?

Reply:
Yes, because it was renamed to secureTransmissionEnabled. It was mentioned in documentation but because it gave an error, I changed it.